test/model_api.py

```diff
@@ -154,10 +154,13 @@ def test_004_local_benchmark(self):
             build_name=build_name,
             rebuild="always",
             cache_dir=cache_dir,
+            lean_cache=True,
```

Comment:
This is now less of a unit test because you're testing two things, both the benchmarking and `lean_cache=True`. Can you make a separate test for the `lean_cache=True` part?

Reply:
Didn't realize the filename! Nevermind!